components/aws/sagemaker/test/integration/utils/minio_utils.py

```diff
@@ -0,0 +1,39 @@
+import utils
```

Comment:
I am not sure but I don't think we should duplicate the code. Can we either re-use it from `test/sample-test/utils.py` or refactor them into something common directory?

Reply:
I think we have to for some parts because the tests will not be running as part of this repositorie's CI and if we depend on master our test infrastructure can become unstable if files or methods change/move